api/src/main/java/org/openmrs/Order.java

```diff
@@ -115,6 +125,19 @@
 	 */
 	private OrderGroup orderGroup;
 	
+	/**
+	 * Represents the status of an order received from a filler 
+	 * @see org.openmrs.Order.FulfillStatus
+	 * @since 2.9.0
+	 */
+	private FulfillStatus fulfillerStatus = FulfillStatus.RECEIVED;
```

Comment:
Do you need a default value? I'd imagine the status would be unknown upon placement of the order

Reply:
+1.  Default should be null.